### PR TITLE
Proofread traditional vaccines

### PR DESCRIPTION
**Pull request by agitter**
Work in progress. I'll finish later today.

### CODE REVIEW

File: content/22.vaccines-trad.md [hunk 1]
```diff
@@ -659,7 +659,7 @@ Overall, the vaccine landscape remains heterogeneous even as the pandemic nears
 The vaccines described in this manuscript, which were developed using well-established technologies, have played a crucial role in improving the feasibility and accessibility of vaccination programs worldwide.
 While the novel technologies have received the bulk of public attention in countries like the U.S., these more traditional vaccine platforms also provide safe and highly effective protection against SARS-CoV-2.
 Although companies developing cutting-edge technologies, namely Moderna and Pfizer/BioNTech, reported very high efficacies greater than 90% in their clinical trials [@individual-vaccines-novel], the efficacies identified in whole-virus and subunit trials have also been very high.
-While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid, 89% [cite]; CoronaVac 83.5% [cite]).
+While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid, 89% [cite]; CoronaVac 83.5% [cite]). <!-- missing two citations here -->
```
Comment (agitter) [inline content/22.vaccines-trad.md:662]: I'm not sure what these citations should be

**Comment by agitter** [comment]
@rando2 I finished my proofreading and this is ready for review. Please note the line that needs two citations because I wasn't sure what to add.

I want to check the latest artifact to confirm my update to the citation of the Manubot version of the manuscript correctly includes the latest authors.

Hopefully it's not too late to send this updated version to the journal (and include my COI from #1193).

**Comment by agitter** [comment]
My fixes in #1196 and #1197 worked and Figure 4 now shows up. I reran the GitHub Actions build and the artifacts there contain the updated artifacts with the figure:
image

I had also updated the manual reference for the Manubot version of the manuscript we use for the online appendix to include the newest authors:
image

`literal` authors aren't being treated correctly though so we get "Medicine II of the IS of" and "Consortium CR". I'm not sure why, because they work for the evolution section of the manuscript.

**Comment by agitter** [comment]
The `literal` authors are working now. The manual reference `id` wasn't correct:
image

File: content/22.vaccines-trad.md (hunk 1 above)
Comment (rando2) [reply content/22.vaccines-trad.md:662]:
```suggestion
While the clinical trial efficacy estimates for IWV and subunit have been lower, some of these trials have also reported efficacies over 80% (e.g., Novavax's Nuvaxovid with 89.7% [@doi:10.1056/NEJMoa2107659] or Sinovac's CoronaVac with 83.5% [@doi:10/gk898z]).
```
Oops, thank you for catching that I forgot to go back and fill these in!

**Comment by agitter** [comment]
> I'll resubmit today.

Thanks! I'll tag v2 of the traditional vaccines manuscript because we should have all the edits now.

Do we want to submit this updated version to arXiv?

**Comment by rando2** [comment]
@agitter Hold off on v2 for a few minutes-- I have a couple of super minor edits made while prepping submission. I can tag you on that PR once everything is final.

**Comment by agitter** [comment]
@rando2 sorry, I was already doing it locally and missed this

I can make a comment on the release about the commits that were actually part of the resubmission or I can retag.

**Comment by rando2** [comment]
Sorry @agitter! It probably isn't a big deal at all -- it's nearly identical, just slight differences in the abstract and then we had lost a table reference somehow.

**Comment by agitter** [comment]
It only takes a minute or two to tag, so I'm okay making v2.1 tags that match what goes to the journal. This time I'll wait until you give me the okay 😄

**Pull request merged**